content/install-ubuntu.md

```diff
@@ -55,7 +55,7 @@ From the menu shown the 'nomodeset' option can be selected during this boot. -->
 
 ![grub](/images/install-ubuntu/GRUB-2.12.png)
 
-Once Ubuntu starts and you have selected your language, any accessibility features that you may need, your keyboard selection, and network connectivity you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**. If you are attempting to restore a broken operating system, please choose **Try Ubuntu**.
+Once Ubuntu starts and you have selected your language, any accessibility features that you may need, your keyboard selection, and network connectivity. Then you will be asked to **Try Ubuntu without installing** or to **Install Ubuntu**. If you are attempting to restore a broken operating system, please choose **Try Ubuntu**.
```

Comment:
I didn't say to change it to `Then you will be asked`, I said `Next, you will be asked`. A comma is needed after an introductory word like that, the way you have it here is incorrect.